packages/components/stories/DatePicker.js

```diff
@@ -83,10 +83,55 @@ storiesOf('DatePicker', module)
 				id="my-date-picker"
 				selectedDateTime={new Date(2018, 4, 13, 12, 30)}
 				onChange={action('onChange')}
+				onFinish={action('onFinish')}
 				name="Datetime"
 			/>
 		</div>
 	))
+	.add('Date picker form mode', () => (
+		<div>
+			<IconsProvider />
+			<h1>DatePicker</h1>
+			<p>
+				Default date picker doesn't include time<br />
+				It comes with classical keyboard gesture.
```

Comment:
Please update the description. Here we don't want to highlight that, but the form mode